packages/modules/web-viewer-react/src/services/Initializer.ts

```diff
@@ -82,15 +41,17 @@ export class WebInitializer {
     if (!IModelApp.initialized && !this._initializing) {
       console.log("starting web viewer");
       this._initializing = true;
+      const iModelAppOptions = getIModelAppOptions(options);
```

Comment:
could we possibily destruct this to get the rpcInterfaces and everything else, where everything else is whats actually used for IModelApp.startup?

and depending on if we remove additionalRpcInterfaces we could remove that, and the reconciliation stuff you have

Reply:
Yes, I think we should remove additionalRpcInterfaces and then can proceed from there to simplify. Thanks